code/game/gamemodes/traitor/double_agents.dm

```diff
@@ -1,14 +1,11 @@
 /datum/game_mode/traitor/double_agents
 	name = "double agents"
 	config_tag = "double_agents"
-	restricted_jobs = list("Cyborg", "AI", "Captain", "Head of Personnel", "Chief Medical Officer", "Research Director", "Chief Engineer", "Head of Security") // Human / Minor roles only.

```

Comment:
After some digging, yes there is an objective that traitors can get to kill an AI

BUT it's not the bog standard assassinate objective, it's the destroy objective (but it still works the same way?)

I'll have to cover the change in objective text for this case, but yeah it can happen.